### PR TITLE
Update test app

### PR DESCRIPTION
**Pull request by ccen-stripe**
# Summary

Update the test app for ID/Address
# Motivation

# Testing

- [ ] Added tests
- [ ] Modified tests
- [x] Manually verified

# Screenshots
| Android  | iOS |
| ------------- | ------------- |
| 3cf5a207-c8d5-4f28-82f7-b4fd992567f6  | simulator_screenshot_E1425299-E462-4B56-A847-3A79DAF897B0 |

### CODE REVIEW

File: tsconfig.build.json [hunk 1]
```diff
@@ -1,5 +1,4 @@
 
 {
-  "extends": "./tsconfig",
-  "exclude": ["example"]
```
Comment (ccen-stripe) [inline tsconfig.build.json:4]: exclude doesn't work in this file, moved to `tsconfig.json`

File: example/src/components/Options.tsx [hunk 2]
```diff
@@ -1,81 +1,153 @@
...
-            onChange={(value) =>
-              setOptions({
-                ...options,
-                allowedTypes: {
-                  ...options.allowedTypes,
-                  [AllowedTypes.ID_CARD]: value,
-                },
-              })
-            }
+  const [verificationTypeRadioButtons] = useState([
```
Comment (ruben-stripe) [inline example/src/components/Options.tsx:18]: This array should be a constant variable instead of a state.

**Review by ruben-stripe — APPROVED** [review]
One small comment

File: example/src/components/Options.tsx (hunk 2 above)
Comment (ccen-stripe) [reply example/src/components/Options.tsx:18]: hmm I tried to replace it with a const, but the UI is not behaving correctly, see below(The [refererence](https://www.npmjs.com/package/react-native-radio-buttons-group) to official API is also using a state)

| Before  | After |
| ------------- | ------------- |
| beforeChagne | afterChange |

**Pull request merged**